posts/2024-04-20-Performance Comparison: Data.table in R and Pandas in Python-Doris Amoakohene/index.qmd

```diff
@@ -0,0 +1,392 @@
+---
+title: 'Performance Comparison: Data.table in R and Pandas in Python'
+author: "Doris Amoakohene"
+date: "2024-03-03"
+output: html_document
+---
+
+# Introduction:
+
+R and Python are two programming languages that have gained immense popularity among data scientists, statisticians, and researchers. 
+In this blog, we will explore two widely used libraries, data.table in R and pandas in Python, which excel in data manipulation and provide versatile functionalities for working with data, focusing on their capabilities for reading, writing, and reshaping data.
+We will also explain how to graphically demonstrate the time taken by each operation. We will use atime package to compare and visualize the asymptotic performance (time and memory usage) of the different functions mentioned above.
+By comparing the asymptotic performance of these packages in these programing languages, we aim to provide insights into their usage and help data scientists make informed choices when it comes to data manipulation and analysis.
```

Comment:
good overall but please add an m in programing

Reply:
thanks, updated